python/util/roles.py

```diff
@@ -44,11 +44,13 @@
 
 ROLE_INTS = {v: k for k, v in ROLE_STRINGS.items()}
 
-BUSINESS_ACCOUNTS = {SALES, DESIGNER, SUPPLIER, MANAGER, ACCOUNTANT, ADMIN}
-
 # users who are in the management team of the domain
 MANAGEMENT_ROLES = {SALES, DESIGNER, MANAGER, ACCOUNTANT, ADMIN}
 
+# users who are involved as business parties
+BUSINESS_ACCOUNTS = MANAGEMENT_ROLES.copy()
```

Comment:
so this will now automagically be changed if someone changes `MANAGEMENT_ROLES`.

it's not super obvious to me whether that's a good or bad thing.